Content.Server/Explosion/Components/TriggerOnProximityComponent.cs

```diff
@@ -0,0 +1,38 @@
+using Robust.Shared.GameObjects;
+using Robust.Shared.Physics.Collision.Shapes;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+using System;
+
+namespace Content.Server.Explosion.Components
+{
+    [RegisterComponent]
+    public class TriggerOnProximityComponent : Component
+    {
+        public override string Name => "TriggerOnProximity";
+        public TimeSpan LastTrigger = TimeSpan.Zero;
+        public string ProximityFixture { get; } = "proximity-fixture";
```

Comment:
i'd like the names to be a bit clearer
```suggestion
        public string FixtureID { get; } = "trigger-on-proximity-fixture";
```